build.go

```diff
@@ -22,7 +23,7 @@ func (ws *workspaceBuilder) PrepareWorkspace(basename string) (string, error) {
 		return "", fmt.Errorf("failed to create workspace directory: %v", err)
 	}
 
-	_, err = cloneRepo(ws.token, workspacePath, ws.clonePath, ws.owner, ws.repo, ws.ref)
+	_, err = cloneRepo(ws.token, workspacePath, ws.clonePath, ws.owner, ws.repo, ws.ref, ws.timeOut)
```

Comment:
The inconsistent capitalization of time-out irks me.  "timeOut" is technically correct but the incorrect form of "timeout" is much more pervasive in software.